### DISCUSSION TITLE
识别增加新的语言

### DISCUSSION THREAD

**Discussion by TyrionChou** [PaddlePaddle/PaddleOCR#12471] [Q&A]
您好，很感谢您的工作，想咨询两个问题
1.可以不使用dict，输入的文本标签可以使用已经转化过后的数字做训练吗？
2.使用dict，空格需要增加在字典里吗，训练需要修改那些代码文件呢？

**Comment by hezhenname** [comment]
你可以看看这个文档recognition.md
